README.md

```diff
@@ -1,26 +1,34 @@
-# Prep Vagrant CentOS 7 VM & deploy on VM to run Create Jazz Lyric app on web
+# Prepare Vagrant CentOS 7 VM to run Create Jazz Lyric Python web app
 
 **Note: Created CentOS 7 machine locally with Vagrant**
 
-1. Copy `setup_machine.sh` with `scp` to the virtual machine/deployment machine, e.g.,<br>
-  `scp -F vagrant-ssh-config setup_machine.sh vagrant@default:setup_machine.sh`
+1. IMPORTANT: This script assumes mysql_pw.txt already exists.<br>
+  Make sure you created mysql_pw.txt outside of this script & put in the password you want for root for login into MySQL on this VM.
 
 2. Create an ssh configuration file, e.g.,<br>
   `vagrant ssh-config > vagrant-ssh-config`
 
-3. Make sure you created mysql_pw.txt outside of this script & put in the password you want for root for login to MySQL. This script assumes mysql_pw.txt already exists.
+3. Copy `setup_machine.sh` & `setup_mysql.sh` with `scp` to the virtual machine/deployment machine, e.g.,<br>
+  `scp -F vagrant-ssh-config setup_machine.sh vagrant@default:setup_machine.sh`<br>
+  `scp -F vagrant-ssh-config setup_mysql.sh vagrant@default:setup_mysql.sh`
```

Comment:
If you want those lines indented in a code block I believe you can do it without the `<br>` tags by using triple back-quotes _and_ indenting the code block at least four spaces. I think this works:

````
2. Copy `setup_machines.sh` & `setup_mysql.sh` with `scp` to the virtual machine/deployment machine
    ```
    scp -F vagrant-ssh-config setup_machine.sh vagrant@default:setup_machine.sh
    scp -F vagrant-ssh-config setup_mysql.sh vagrant@default:setup_mysql.sh
    ```
````

Reply:
Done. That is better with 4 spaces to indent. Atom changed the 2 spaces I added to the end of lines for a new line to those <br> tags.